src/marqo/tensor_search/tensor_search.py

```diff
@@ -811,7 +861,7 @@ def _add_documents_structured(config: Config, add_docs_params: AddDocsParams, ma
                         if (
                                 add_docs_params.mappings is not None and
                                 field_name in add_docs_params.mappings and
-                                add_docs_params.mappings[field_name]["type"] == "multimodal_combination"
+                                add_docs_params.mappings[field_name]["type"] == FieldType.MultimodalCombination
```

Comment:
Does this work? I think you need something like `FieldType.MultimodalCombination.value`?